cukinia

```diff
@@ -25,6 +25,10 @@ GENERIC_FUNCTS='cmd
 # By default log to stdout
 __log_file="/dev/stdout"
 
+# Default temporary files
+__stderr_tmp='stderr.tmp'
+__stdout_tmp='stdout.tmp'
```

Comment:
you should use non-local files as we have no warranty if the cwd is writable. the return if $(mktemp) is fine, don't forget to clean them up